pkg/config/types.go

```diff
@@ -70,7 +70,7 @@ type (
 
 	CLI struct {
```

Comment:
Can you just also, change this to 'Client'? I think CLI is an outdated name for us.